plugins/monorepo/src/load-workspace-configs.ts

```diff
@@ -0,0 +1,58 @@
+import { Init, InitContext } from '@dotcom-tool-kit/base'
+import path from 'path'
+import fs from 'fs/promises'
+import { ValidConfig } from '@dotcom-tool-kit/config'
```

Comment:
**todo**: this and `@dotcom-tool-kit/base` in `workspace-command` aren't included in the `package.json`. the `tsconfig.json` doesn't match up either